bin/invalidate-project-configs

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+from sentry.runner import configure
+
+
+def main():
+    """Gradually evict existing project configs from redis and recompute them.
+
+    This can be used to speed up incident recovery, for example when faulty project configs
+    that cannot be read by Relay have been written to redis.
+    """
+    configure()
+    from sentry.models import Organization
+    from sentry.tasks.relay import schedule_invalidate_project_config
+    from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+    # NOTE: Instead of iterating over all projects, we could use redis.scan_iter()
+    # to evict only project configs that exist in redis.
+    queryset = Organization.objects.all().values_list("id", flat=True)
+    print("Invalidating all project configs...")  # NOQA S002
+    for org_id in RangeQuerySetWrapperWithProgressBar(queryset, result_value_getter=lambda x: x):
```

Comment:
This fetches all organizations in batches of 1000, which is the slow-but-stable way of iterating that we also use in migrations.